README.md

```diff
@@ -1,2 +1,169 @@
-# Decode-a-Morse-code-message
-This project is about using Ruby code to decode a Morse code message
+<a name="readme-top"></a>
+<div align="center">  
+  <h2><b>Ruby Methods to Decode a Morse Message</b></h2>
+</div>
+<div align="center">
+  <img src="./Morse_code.png" alt="Morse code"/>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  <!-- - [Run tests](#run-tests) -->
```

Comment:
- [ ] It would be nice if you fix the Install and Usage links because these are broken 😨 Kindly fix them or remove them to make your readme descriptive and professional 👍